install/Dockerfile.hdidx

```diff
@@ -0,0 +1,21 @@
+FROM ann-benchmarks
+
+# needed to avoid some dependencies starting interaction on the command line
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    python-opencv \
+    python-numpy \
+    python-pip \
+    git
+RUN pip install cython
+RUN pip install -r requirements.txt
+
+# download from aagnone's fork, which fixes some type-casting bugs
+RUN git clone https://github.com/aagnone3/hdidx.git
```

Comment:
I guess it would be nice to get this merged back into hdidx so that we’re testing the “official” version, but no rush

Reply:
Agreed, I will PR to the owner